idd/Energy+.idd.in

```diff
@@ -82147,6 +82147,233 @@ ThermalStorage:ChilledWater:Stratified,
        \units W/K
        \default 0.0
 
+ThermalStorage:HotWater:Stratified,
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \reference-class-name validPlantEquipmentTypes
+       \reference validPlantEquipmentNames
+       \reference-class-name validCondenserEquipmentTypes
+       \reference validCondenserEquipmentNames
+       \reference-class-name validBranchEquipmentTypes
+       \reference validBranchEquipmentNames
+  N1 , \field Tank Volume
+       \required-field
+       \type real
+       \units m3
+       \minimum> 0.0
+       \ip-units gal
+  N2 , \field Tank Height
+       \required-field
+       \type real
+       \units m
+       \minimum> 0.0
+       \note Height is measured in the axial direction for horizontal cylinders
+  A2 , \field Tank Shape
+       \type choice
+       \key VerticalCylinder
+       \key HorizontalCylinder
+       \key Other
+       \default VerticalCylinder
+  N3 , \field Tank Perimeter
+       \type real
+       \units m
+       \minimum 0.0
+       \note Only used if Tank Shape is Other
+  A3 , \field Top Setpoint Temperature Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+  A4 , \field Bottom Setpoint Temperature Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+  N4 , \field Deadband Temperature Difference
+       \type real
+       \units deltaC
+       \minimum 0.0
+       \default 0.0
+  N5,  \field Top Temperature Sensor Height
+       \units m
+       \minimum 0.0
+  N6,  \field Bottom Temperature Sensor Height
+       \units m
+       \minimum 0.0
+  N7 , \field Maximum Temperature Limit
+       \type real
+       \units C
+  N8 , \field Nominal Heating Capacity
+       \type real
+       \units W
```

Comment:
This isn't autosizable?

Reply:
So I was following the "Nominal Cooling Capacity" field in the `ThermalStorage:ChilledWater:Stratified`. In that object, this capacity field is used for supervisory control where plant operation schemes require a cooling capacity to model how equipment is dispatched.